apps/meteor/tests/data/rooms.helper.js

```diff
@@ -65,3 +65,27 @@ function actionRoom({ action, type, roomId }) {
 export const deleteRoom = ({ type, roomId }) => actionRoom({ action: 'delete', type, roomId });
 
 export const closeRoom = ({ type, roomId }) => actionRoom({ action: 'close', type, roomId });
+
+export const setDefaultRoom = ({ roomId }) => {
+	return request
+		.post(api('rooms.saveRoomSettings'))
+		.set(credentials)
+		.send({
+			rid: roomId,
+			default: true,
+			favorite: {
+				defaultValue: true,
+				favorite: false
+			}
```

Comment:
We should not favorite the room here since the function name is saying it's just making it as default. I think it would be better to have two different functions or create a saveRoomSettings function and accept the desired parameters.